python/wpt/grouping_formatter.py

```diff
@@ -170,6 +170,9 @@ def test_end(self, data: dict) -> Optional[UnexpectedResult]:
         test_path = data["test"]
         del self.running_tests[data['thread']]
 
+        if test_status == "SKIP":
+            print(data["message"])
+
```

Comment:
This handler isn't meant to print any messages. I think this will cause issues with the `ServoFormatter` which extends it since it expects to be able to constantly erase and rewrite the last terminal lines. Probably the right way to go about this is to record the skipped reason here and then print it like stack trace output in `ServoFormatter`.

Reply:
With the latest changes, this is the output when running multiple tests:
```
Ran 9 tests finished in 0.0 seconds.
  • 9 ran as expected. 9 tests skipped.
   Executor does not support testdriver.js:
     /cookies/attributes/attributes-ctl.sub.html
     /cookies/attributes/domain.sub.html
     /cookies/attributes/expires.html
     /cookies/attributes/invalid.html
     /cookies/attributes/max-age.html
     /cookies/attributes/path-redirect.html
     /cookies/attributes/path.html
     /cookies/attributes/secure-non-secure.html
     /cookies/attributes/secure.https.html
```
But I'm not clear on how. I can modify the output for running a single test, since ServoFormatter does not appear to be used.